### DISCUSSION TITLE
Welcome to endo Discussions!

### DISCUSSION THREAD

**Discussion by michaelfig** [endojs/endo#1219] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kriskowal** [comment]
👋 Hello, I work on Endo at Agoric. I picked up module support for compartments where @michaelfig and @erights had started and built out the compartment mapper, to prove that Compartments are suitable to emulate Node.js, to replace risky dependencies in the hardened JavaScript shim (`ses`) and Agoric SDK on Rollup, and provide a foundation for LavaMoat to support ECMAScript module loading. My biggest Endo priority is to build out an Endo user agent for peer-to-peer application sandboxes, like Chrome extensions but p2p, to help @naugtur advance LavaMoat, and to coordinate the Compartment design with @patrick-soquet and @phoddie who develop the XS JavaScript engine. I’m busy championing [Compartments as a feature of JavaScript](https://github.com/tc39/proposal-compartments/).
